packages/common/src/enums.ts

```diff
@@ -25,6 +25,7 @@ export enum Hardfork {
   Merge = 'merge',
   Eof = 'eof',
   Shanghai = 'shanghai',
+  ShardingFork = 'shardingFork',
```

Comment:
I think I am still not happy that we are mixing the super-official mainnet forks like `Shanghai` with our own inventions so transparently here and I wouldn't want to release it this way.

Can we rename this to something which clearly separates, e.g. `Eof_INTERNAL`, `ShardingFork_INTERNAL`, so that this is not sneaking in into peoples production code somehow and will unnecessarily break their code if we change here?

Reply:
Done.  I also removed the shardingFork reference from the EVM list of supported hardforks for now.  It will stay in my 4844 branch for now.